oscap tests -  cls.fetch_scap_and_profile_id throws TypeError

```
        cls.scap_id_rhel6, cls.scap_profile_id_rhel6 = (
            cls.fetch_scap_and_profile_id(
                cls.title,
>               OSCAP_PROFILE['common']
            )
        )

tests/foreman/cli/test_oscap.py:117: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/foreman/cli/test_oscap.py:92: in fetch_scap_and_profile_id
    for profile in default_content['scap-content-profiles']
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

.0 = <dict_keyiterator object at 0x7f36af561c78>

    profile['id']
    for profile in default_content['scap-content-profiles']
>   if scap_profile in profile['title']
            ]
E           TypeError: string indices must be integers

tests/foreman/cli/test_oscap.py:93: TypeError
```

--- a/tests/foreman/cli/test_oscap.py
+++ b/tests/foreman/cli/test_oscap.py
@@ -85,7 +85,9 @@ class OpenScapTestCase(CLITestCase):
 
         :returns: scap_id and scap_profile_id
         """
-        default_content = Scapcontent.info({'title': scap_name})
+        default_content = Scapcontent.info({'title': scap_name},
+                                           output_format='json'
+                                           )
         scap_id = default_content['id']
         scap_profile_ids = [
             profile['id']

